de.fu_berlin.inf.dpp/src/de/fu_berlin/inf/dpp/editor/DirtyStateListener.java

```diff
@@ -54,6 +55,9 @@ public void elementDirtyStateChanged(Object element, boolean isDirty) {
 
         final IFile file = ((FileEditorInput) element).getFile();
 
+        final IReferencePointManager referencePointManager = editorManager
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "referencePointManager" local variable. ([squid:S1481](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS1481))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=274&issues=AWaXS6DGYAGQNfrBSSyX&open=AWaXS6DGYAGQNfrBSSyX)

[ISSUE_KEY]: AWaXS6DGYAGQNfrBSSyX